### PR TITLE
Admin Intercom Settings

### PR DESCRIPTION
**Pull request by shivamvinsol**

### CODE REVIEW

File: app/controllers/spree/admin/intercom_controller.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class Spree::Admin::IntercomController < Spree::Admin::BaseController
```
Comment (sawangupta92) [inline app/controllers/spree/admin/intercom_controller.rb]: inherit from resourcecontroller

File: app/views/spree/admin/intercom/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<% content_for :page_title do %>
```
Comment (sawangupta92) [inline app/views/spree/admin/intercom/edit.html.erb]: fix design as discussed

File: config/routes.rb [hunk 3]
```diff
@@ -1,3 +1,6 @@
 Spree::Core::Engine.add_routes do
   # Add your extension routes here
+  namespace :admin do
+    resource :intercom, only: [:edit, :update], controller: 'intercom'
```
Comment (sawangupta92) [inline config/routes.rb]: can we use inflection here?

**Review by sawangupta92 — CHANGES_REQUESTED** [review]

File: spree_intercom.gemspec [hunk 4]
```diff
@@ -37,4 +37,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'mysql2'
   s.add_development_dependency 'sqlite3'
   s.add_development_dependency 'appraisal'
+  s.add_development_dependency 'shoulda-matchers'
```
Comment (tanmay3011) [inline spree_intercom.gemspec:40]: @shivamvinsol Please change email and homepage to
 ``` s.email     = 'info@vinsol.com'```
``` s.homepage  = 'http://vinsol.com'```

**Pull request merged**